liberty/base.go

```diff
@@ -100,6 +140,29 @@ func (b Base) ContributeExternalConfiguration(layer libcnb.Layer) error {
 	if err := util.Extract(artifact, confPath, c); err != nil {
 		return fmt.Errorf("unable to expand external configuration\n%w", err)
 	}
+	
+	fixes, error := ioutil.ReadDir(filepath.Join(confPath, "ifixes"))
```

Comment:
```suggestion
	fixes, err := ioutil.ReadDir(filepath.Join(confPath, "ifixes"))
```

Using `err` for the name of the returned err is convention.